public/app/features/alerting/components/AlertingQueryPreview.tsx

```diff
@@ -40,30 +43,60 @@ export const AlertingQueryPreview: FC<Props> = ({ queryRunner }) => {
         })}
       </TabsBar>
       <TabContent className={styles.tabContent}>
-        {activeTab === Tabs.Query && data && (
-          <div>
-            <Table data={data.series[0]} width={1200} height={300} />
+        {data && data.series.length > 0 ? (
+          <AutoSizer style={{ width: '100%', height: '100%' }}>
+            {({ width, height }) => {
+              switch (activeTab) {
+                case Tabs.Instance:
+                  return <PreviewInstancesTab isTested={false} data={data} styles={styles} />;
+
+                case Tabs.Query:
+                default:
+                  return <PreviewQueryTab data={data} width={width} height={height} />;
+              }
+            }}
+          </AutoSizer>
+        ) : (
+          <div className={styles.noQueries}>
+            <h4 className={styles.noQueriesHeader}>No queries added.</h4>
+            <div>Start adding queries to this alert and a visualisation for your queries will appear here.</div>
+            <div>
+              Learn more about how to create alert definitions <Icon name="external-link-alt" />
+            </div>
           </div>
```

Comment:
nit: maybe could also be extracted to a smaller component like PreviewInstancesTab?